content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -116,3 +116,29 @@ Here are some helpful resources to get started with `policy-controller`:
 
 - [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller)
 - [Configuration Options](https://github.com/sigstore/policy-controller/tree/main/config)
+
+## Verify the Software Bill Of Materials (SBOM)
+
+The Kubernetes SBOM is signed as well and can be verified by using the sigstore
+certificate and signature or the corresponding SHA files:
```

Comment:
Nit: I think this would improve readability. I hope I got it correctly.
```suggestion
certificate and signature, or the corresponding SHA files:
```

Reply:
Fixed as suggested.